### PR TITLE
Make nengo.ai friendlier for first timers

### PR DESCRIPTION
**Pull request by tbekolay**
This PR makes a few changes to improve the experience for people completely new to Nengo.

- Rename Download to Getting started
- Reorganize the Getting started page a bit to hide detailed install stuff so that the "how to run" elements are more prominent
- Add a nengo.ai/examples page that points to a bunch of examples, grouped in rough categories
- Remove the nengo-examples project from the Docs dropdown as it hasn't been built out much yet

This PR also adds the ABR logo to the footer.

### CODE REVIEW

File: _includes/install-info.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<h2>
+  <a
+    href="#"
+    class="collapsed"
+    data-toggle="collapse"
+    data-target="#installing-python-and-numpy"
+  >
```
Comment (krisfrenette) [inline _includes/install-info.html]: The icon for collapsing and expanding aren't showing for me.

Comment (tbekolay) [reply _includes/install-info.html]: Hm, it's supposed to be using a fontawesome icon, which is included in the `default.html` layout, do you see a `use.fontawesome.com` stylesheet link in the page's `<head>`?

We could switch to doing the caret using CSS, it looks like the dropdowns in the header (e.g. after "Documentation") are drawn with CSS, maybe we can just reuse that class instead.

Comment (krisfrenette) [reply _includes/install-info.html]: Looking at this more, should we just remove the collapse? The tabbed info isn't that tall and people can just scroll by if they want?

Comment (tbekolay) [reply _includes/install-info.html]: Yeah it's definitely not as critical having put the install stuff into tabs... I do still kind of like having this collapsed, but I'm fine either way. @drasmuss any preference?

Comment (krisfrenette) [reply _includes/install-info.html]: I see the font-awesome but still just a box. Not sure what is going on. I don't have strong feelings on removing the collapse. Likely just a build issue, so I am good either way.

Comment (tbekolay) [reply _includes/install-info.html]: Interesting. We'll see what Dan says about the collapsing and if we still want it collapsed I'll switch it to using CSS to draw the caret.

**Pull request merged**